Kudu.Services.Web/Content/Scripts/GeneralModal.js

```diff
@@ -0,0 +1,58 @@
+
+
+
+var generalModal = {
+    _createIfNotExistant: function () {
+        var popup = $("#generalModal");
+       // console.dir(popup);
```

Comment:
might want to do some cleanup, see a few commented console.log/console.dir/alert